.gitpod.yml

```diff
@@ -0,0 +1,5 @@
+# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
```

Comment:
?

Reply:
Thats a configuration file for gitpod, it’s not exactly necessary but its helpful for people who want to use gitpod to contribute.